core/src/main/resources/logback.xml

```diff
@@ -19,7 +19,7 @@
             <level>INFO</level>
         </filter>
         <rollingPolicy class="ch.qos.logback.core.rolling.TimeBasedRollingPolicy">
-            <fileNamePattern>./logs/hera.%d{yyyy-MM-dd}.log</fileNamePattern>
+            <fileNamePattern>logs/hera.%d{yyyy-MM-dd}.log</fileNamePattern>
```

Comment:
I read that logback sometimes acts up when specifying relative file paths, so I removed them